templates/deploy-ansible-service-broker.template.yaml

```diff
@@ -215,8 +292,11 @@ objects:
           white_list:
             - ".*-apb$"
       dao:
-        etcd_host: 0.0.0.0
+        etcd_host: asb-etcd.${NAMESPACE}.svc
```

Comment:
Why do you need ```.${NAMESPACE}.svc```?

Reply:
Using `${NAMESPACE}` will allow helios broker to pass the aws-service-broker namespace and use the same config. I did not go back and fix every instance but don't see why we can't start now. I can remove it if you feel strongly.

Reply:
irc discussion: https://docs.openshift.com/online/dev_guide/secrets.html#service-serving-certificate-secrets